collector/zfs.go

```diff
@@ -35,9 +35,25 @@ type zfsSubsystemName string
 
 const (
 	arc            = zfsSubsystemName("zfsArc")
+	dmuTx          = zfsSubsystemName("zfsDmuTx")
+	fm             = zfsSubsystemName("zfsFm")
+	vdevCache      = zfsSubsystemName("zfsVdevCache")
+	xuio           = zfsSubsystemName("zfsXuio")
+	zfetch         = zfsSubsystemName("zfsFetch")
+	zil            = zfsSubsystemName("zfsZil")
 	zpoolSubsystem = zfsSubsystemName("zfsPool")
 )
 
+var subsystems = []zfsSubsystemName{
```

Comment:
Glad you don't mind nitpicking.. because here is another one :)
No need for this to be a global variable. Can you define the list inline in Update? Like:

```
for _, subsystem := range []zfsSubsystemName{
    arc,
    ...
```

Or is this used somewhere else? In general, if global variables are necessary they should have some prefix because all collectors share the same namespace. But I think here we don't need a variable at all.